datasets/_externals/ext_gb_fca_firds.yml

```diff
@@ -0,0 +1,83 @@
+type: external
+title: UK Financial Instruments Reference Data System (FIRDS)
+entry_point: zavod.runner.local_enricher:enrich
+full_dataset: gb_fca_firds
+disabled: false
+hidden: false
+coverage:
+  frequency: weekly
+  start: 2025-06-02
+exports:
+  - statistics.json
+  - entities.delta.json
+deploy:
+  cpu: "1000m"
+  cpu_limit: "2000m"
+  memory: "5000Mi"
+  memory_limit: "6000Mi"
+  disk: "100Gi"
+  disk_limit: 100Gi
+load_statements: true
+ci_test: false
+summary: >-
+  List of financial instruments and their reference data, as published by the
+  Financial Conduct Authority (FCA) in the UK.
+description: |
+  This is a dataset of financial instruments and their reference data, as published
+  by the Financial Conduct Authority (FCA) in the UK. The data establishes
+  connections between financial instruments and their issuers, and includes
+  information such as the instrument's name, ISIN, and LEI.
+url: https://data.fca.org.uk/#/download
+publisher:
```

Comment:
maybe nice to duplicate the related dataset data section for now to make it easy to find. I see ext_eu_esma_firds does that.